2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,12 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+     let newArr = [];
+   for (let title of allArticleTitles) {
+     if (title.length < 65) newArr.push(title);
```

Comment:
Just to keep things readable, I'd advise always using `{}` with `if` statements even if they're 1 line:
```
if(title.length < 65) {
   newArr.push(title)
}
```